bgpd/bgp_bmp.c

```diff
@@ -1120,6 +1120,9 @@ static bool bmp_wrsync(struct bmp *bmp, struct pullwr *pullwr)
 		bmp_monitor(bmp, adjin->peer, 0, bn_p, prd, adjin->attr, afi,
 			    safi, adjin->uptime);
 
+	if (bn)
```

Comment:
I see several places we return after the lookup before this spot.  They should be fixed too?

Reply:
Seems it's not necessary, because `bn` is NULL there, right?

Reply:
ok got it